packit/constants.py

```diff
@@ -89,6 +89,13 @@
 downstream_package_name: {downstream_package_name}
 """
 
+PRECOMMIT_CONFIG = {
+    # check validity of Packit configuration
+    "repo": "https://github.com/packit/pre-commit-hooks",
+    "rev": "v1.2.0",
```

Comment:
This might bite us in the future, but I don't think there is a reasonable way how to handle this…

Reply:
It will, but the UX still sucks

1. Run `packit init`
2. Run `pre-commit autoupdate`?

:/ I don't think there is a reasonable way to handle this. Everything else would just overcomplicate.